lib/ansible/modules/web_infrastructure/jenkins_plugin.py

```diff
@@ -120,6 +120,8 @@
         manually.
       - It might take longer to verify that the correct version is installed.
         This is especially true if a specific version number is specified.
+      - Put the value under quotes so that values like '1.2.30' are not parsed
+        incorrectly as 1.2.3
```

Comment:
The example here is incorrect as the `1.2.30` will not be interpreted as `1.2.3`. Please use the example from the description of the issue where `1.20` is interpreted as `1.2`.

Please change the text to this:

```
      - Quote the version to prevent the value to be interpreted as float. For
        example if C(1.20) would be unquoted, it would become C(1.2).
```